content/posts/optimize_expressjs_apps.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Ultimate Guide to Optimizing Website and Application Development with Express.js
+date: 2024-08-22
+tags:
+  - Learning
+author: Bogdan Ripa
+linkedIn: https://www.linkedin.com/in/bogdanripa
+thumbnail: /images/fosdem-2024.webp
```

Comment:
We have to change this to use a branding-aligned thumbnail image.

Reply:
how?